app/ante_handler.go

```diff
@@ -7,17 +7,22 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
```

Comment:
[ante.go](https://github.com/CosmosContracts/juno/blob/v2.3.0/app/ante.go) is used as a reference for `ante_handler.go`